pkg/kubelet/container_bridge.go

```diff
@@ -43,15 +43,16 @@ func createCBR0(wantCIDR *net.IPNet) error {
 		glog.Error(err)
 		return err
 	}
-	// restart docker
+	// Stop docker so that babysitter process can restart it again with proper configurations and
+	// checkpoint file
 	// For now just log the error. The containerRuntime check will catch docker failures.
 	// TODO (dawnchen) figure out what we should do for rkt here.
 	if util.UsingSystemdInitSystem() {
-		if err := exec.Command("systemctl", "restart", "docker").Run(); err != nil {
+		if err := exec.Command("systemctl", "stop", "docker").Run(); err != nil {
```

Comment:
Like you mentioned, upstart and systemd are valid babysitters. If someone is running one of those, they *don't* need another babysitter right?

My desktop uses upstart, and `service docker stop` will not bring it back. This is inline with my understand of modern init systems, but I'm no expert. So most of my confusion is around how a "real" init system will handle a stop command, can you clarify?